selfapi.js

```diff
@@ -128,6 +133,9 @@ API.prototype = {
     // Export own request handler examples as test functions.
     for (var method in self.handlers) {
       var handler = self.handlers[method];
+      if (!handler.examples) {
+        throw `Handler ${method.toUpperCase()} ${this.path} does not have examples!`;
```

Comment:
Nit: We purposefully kept the code compatible with ES < 6, please don't use string templates.